v2/app/collegevine-hub/CVHubViewer.jsx

```diff
@@ -2036,6 +2036,10 @@ var PivotViewer = (Pivot.PivotViewer = function(
     rearrange()
   }
 
+  this.setPadding = function(padding) {
+    this.padding = { ...this.padding, ...padding }
+  }
```

Comment:
Although we can use modern JS, we still have to be careful with `self` vs `this`. For consistency with the other methods, can we use `...self.padding` inside the function? With the old-skool inheritance, etc. I’m worried it might break in certain cases since `this` is dynamic. Alternatively, we could do `this.setPadding = padding => { … }` since arrow functions preserve lexical `this`.